### ISSUE TITLE
Loscil not reading tables loaded in with readbuf in csound~

### ISSUE THREAD

**Issue by unriginal** [csound/csound#502]
If I load a sample into an ftable in csound~ using the readbuf message, loscil will not play it. If I use writebuf to copy the ftable back into a buffer in Max, it works as expected- copying the entire ftable into the buffer.
Example csd, maxpat, and sample:
https://www.dropbox.com/s/jcivqb5y18bfijv/loscilLoadTest.zip?dl=0

**Comment by jpffitch** [comment]
When you say youwrote an ftable wirg readbuf can you give more detail? Like how did you obtain the addreess for the data; who allocated te space etc.

**Comment by unriginal** [comment]
Sure. When you send csound~ a readbuf message, it takes one channel of a buffer that has already been allocated by Max, and copies it into an already allocated ftable in Csound. It either copies as many samples as the ftable can hold, or resizes the ftable to be as big as the buffer (In my case, the latter). It handles all of this internally, so I don't know what the addresses are. The reverse operation is writebuf, and when I use it on the ftable that I've copied into with a readbuf message, I get a buffer that is identical to the one that I used readbuf on.
